### PR TITLE
Include full describe heirarchy in file name

### PR DESCRIPTION
**Pull request by aldenquimby**
Test example:
```
describe('AHCD Flow Happy Path', () => {
  ...
  describe('AHCD Review page', () => {
    ...
    it('matches visually', () => {
      cy.matchScreenshot();
    });
    ...
  });
  ...
});
```

Before this PR, fileName = `AHCD Review page -- matches visually.png`

After this PR, hoping it's `AHCD Flow Happy Path -- AHCD Review page -- matches visually.png`

This should make our screenshot organization quite a bit cleaner

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -111,23 +109,29 @@ const copyToNewDir = (fileName, newImage, cb) => {
...
+};
+
 /**
  * Takes a screenshot and, if available, matches it against the screenshot
  * from the previous test run. Assertion will fail if the diff is larger than
  * the specified threshold
- * @param  {String} name
  * @param  {Object} options
  */
-function matchScreenshot(
```
Comment (aldenquimby) [inline src/index.js:121]: Moved `name` into options to make usage like this cleaner: https://github.com/FreeWillPBC/freewill-api-v2/blob/0f73add0b429b74b16630be787ce6e22d7a0ca61/src/web/cypress/integration/qcd.spec.js#L85-L90

File: src/screenshotMatches.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
-module.exports = function({ oldImage, newImage, target, threshold, thresholdType }, cb) {
```
Comment (aldenquimby) [inline src/screenshotMatches.js:1]: Just eslint changes in this file

File: src/task.js [hunk 3]
```diff
@@ -1,41 +1,43 @@
+/* eslint-disable no-unused-vars */
```
Comment (aldenquimby) [inline src/task.js:1]: Just eslint changes in this file

**Review by jboolean — APPROVED** [review]
Nice! That wasn't so bad.

**Pull request merged**